### DISCUSSION TITLE
Mobile Blazor Bindings vs Net maui blazor

### DISCUSSION THREAD

**Discussion by whann0205** [dotnet/maui#3920] [General]
Is that project dead and everybody should migrate to net maui blazor?
Can net maui still make hybrid applications with native and blazor type controls?
Why does mobile blazor bindings 0.5.x is not working anymore on ios
devices?
Is it kind of force to move to net maui?

**Comment by Velobook** [comment]
You know what? I hate MAUI . I love BLAZOR :-)

**Reply by PirasathLuxchumykanthan** [reply]
So true..
